types/badwords/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for badwords 1.0
+// Project: https://github.com/MauriceButler/badwords#readme
+// Definitions by: Maurice Butler <https://github.com/MauriceButler>
```

Comment:
```ts
// Definitions by: Maurice Butler <https://github.com/MauriceButler>
```
you have to get this person explicit ACK to be listed here, otherwise just give the person a credit in a file. The person handle will receive notifications